project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -33,7 +34,7 @@ export default class HeatMap extends React.Component {
       this.cancelQueuedProcess();
       this.refreshTimerId = setTimeout(this.asyncInvokeDashboardRefresh(viewport), constants.MAP.DEBOUNCE);
     }
-
+    this.setState({mapMovedByProps: false});
```

Comment:
`mapMovedByProps` is never set to true so might as well remove it. Same on L56 below.